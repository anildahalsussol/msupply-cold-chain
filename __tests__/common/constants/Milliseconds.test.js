import { MILLISECONDS } from '~constants';

describe('Constants: MILLISECONDS', () => {
  it('Matches snapshot', () => {
    expect(MILLISECONDS).toMatchSnapshot();
  });
});
