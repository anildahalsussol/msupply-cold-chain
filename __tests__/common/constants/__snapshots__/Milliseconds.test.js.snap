// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: MILLISECONDS Matches snapshot 1`] = `
Object {
  "ONE_DAY": 86400000,
  "ONE_MINUTE": 60000,
  "ONE_SECOND": 1000,
  "QUARTER_SECOND": 250,
  "SIXTY_SECONDS": 60000,
  "TEN_MINUTES": 600000,
  "TEN_SECONDS": 10000,
  "THIRTY_MINUTES": 1800000,
  "THIRTY_SECONDS": 30000,
}
`;
